src/v2/Apps/Order/Components/OfferHistoryItem.tsx

```diff
@@ -29,14 +31,14 @@ const OfferHistoryItem: React.FC<
             : "Your offer"}
         </Text>
         <Text variant={["xs", "sm"]} color="black100">
-          {lastOffer?.amount}
+          {appendCurrencySymbol(lastOffer?.amount, currency)}
```

Comment:
To avoid duplicating logic in many places and help consolidate the display across all surface areas, it will be great the implementation is in Metaphysics and clients can just query for it. For example, we can expand the [`amount`](https://github.com/artsy/metaphysics/blob/5e9eb1795fd261ecb8f94ad76a56e94436f56870/src/schema/v1/fields/money.ts#L28-L71) helper to consider displaying the currency code.

In parallel, I think design is working on [consolidating money displays](https://artsy.slack.com/archives/C9XJKPY9W/p1635442601002500) so we have some standard to follow.